src/pages/rules/Rules.js

```diff
@@ -0,0 +1,26 @@
+import { Link } from "react-router-dom";
+import { NavBar } from "../../components/NavBar/NavBar";
+import "./rules.css";
+
+const Rules = () => {
+    return (
+        <div class="page-layout">
+            <NavBar />
+            <main class="rules-main flex flex-center">
+                <div class="rules-body flex flex-column flex-gap-1 flex-space-between">
+                    <h3>Rules</h3>
```

Comment:
make a JSON file for Rules So in the future if you want to change anything you just change in JSON file